src/main/java/TypeAccuracy.java

```diff
@@ -0,0 +1,78 @@
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Calculates the accuracy of the user's input against the test text
+ */
+public class TypeAccuracy {
+
+    private static Logger logger = Logger.getLogger("TypeAccuracyLogger");
+
+    static {
+        logger.setLevel(Level.WARNING);
+    }
+
+    private ArrayList<String> testText;
+    private ArrayList<String> userText;
+
+    /**
+     * Constructs a TypeAccuracy object used to find the typing of the user's input
+     * @param userText ArrayList containing the user's input line by line
+     */
+    public TypeAccuracy(ArrayList<String> userText) {
+        this.userText = userText;
+        logger.log(Level.INFO, "TypeAccuracy object successfully created. UserText initialized.");
+    }
+
+    /**
+     * Sets the test text to the text randomly selected by the text selector class
```

Comment:
nitpick, but following the sample javadoc guides, should send the comment with a fullstop
https://nus-cs2113-ay2425s2.github.io/website/se-book-adapted/chapters/documentation.html#javadoc